src/battery_manager_laptop.cpp

```diff
@@ -0,0 +1,66 @@
+#include <ros/ros.h>
+#include <sensor_msgs/BatteryState.h>
+#include <std_msgs/String.h>
+
+int percentage;
+
+void batteryCallback(const sensor_msgs::BatteryState::ConstPtr& msg)
+{
+    double capacity;
+    double charge;
+
+    if (msg->location == "hero2")
```

Comment:
Can't we make this more general?
E.g. have the robot say 'The battery in X is almost empty' with X being `msg->location`